e2e/content-services/version/version-permissions.e2e.ts

```diff
@@ -40,11 +40,11 @@ import CONSTANTS = require('../../util/constants');
 describe('Version component permissions', () => {
 
     const loginPage = new LoginPage();
-    const contentListPage = new ContentListPage();
     const versionManagePage = new VersionManagePage();
     const navigationBarPage = new NavigationBarPage();
     const uploadDialog = new UploadDialog();
     const notificationPage = new NotificationPage();
+    const contentServices = new ContentServicesPage();
```

Comment:
can we name the 'contentServices' as 'contentServicesPage'

Reply:
Do we really need to ?